src/services/nexboard/services/Boards.js

```diff
@@ -10,7 +11,12 @@ class Board {
 		return nexboardClient.getBoard(id);
 	}
 
-	create({ projectId, title = 'Neues Nexboard Board', description = 'Ein digitales Whiteboard' }, params) {
+	create({ title = 'Neues Nexboard Board', description = 'Ein digitales Whiteboard' }, params) {
```

Comment:
how about language keys for this? not sure though about the usage of them in old server

Reply:
Since we have no lang keys in server, nothing we can do about it. However, is nothing changed. Normally, the values are set when calling the service, with the title and description given from the user